docs/content/docs/connectors/datastream/elasticsearch.md

```diff
@@ -42,16 +42,12 @@ of the Elasticsearch installation:
     </tr>
   </thead>
   <tbody>
-    <tr>
-        <td>5.x</td>
-        <td>{{< artifact flink-connector-elasticsearch5 >}}</td>
-    </tr>
     <tr>
         <td>6.x</td>
```

Comment:
Don't we support only a max version for 6.x?

Reply:
You are right, I'll fix that